### ISSUE TITLE
Whether pretrained model in this repo is trained with VGG face2 dataset?? 

### ISSUE THREAD

**Issue by RaviRaaja** [davidsandberg/facenet#637]
Hi ,
I'm bit curious to know whether pretrained models available in this repo has been trained using 3.31 Million images provided from vgg face2 (http://www.robots.ox.ac.uk/~vgg/data/vgg_face2/) ,if yes then i will use this embeddings ____ if not then , iam thinking to train available models on this dataset so I need some suggestions for model architectures and hyperparameter selection !

**Comment by Shahnawazgrewal** [comment]
yes you can use it. I already trained a model with center loss on vggface2

**Comment by SiddhardhaSaran** [comment]
Where can I find the model you trained?@Shahnawazgrewal

**Comment by Zumbalamambo** [comment]
[Refer](https://github.com/davidsandberg/facenet/issues/624) he will upload and share...

**Comment by RaviRaaja** [comment]
Thanks

**Comment by Shahnawazgrewal** [comment]
Validation on LFW dataset with the model trained on VGGFace2
Runnning forward pass on LFW images
Model directory: /home/super/datasets/lfw/vggface2-cl
Metagraph file: model-20171216-232945.meta
Checkpoint file: model-20171216-232945.ckpt-100000
Runnning forward pass on LFW images
Accuracy: 0.992+-0.004
Validation rate: 0.96000+-0.01880 @ FAR=0.00067
Area Under Curve (AUC): 0.999
Equal Error Rate (EER): 0.008
